### DISCUSSION TITLE
how to update

### DISCUSSION THREAD

**Discussion by eztitles** [polymind-inc/acmebot#661] [Q&A] [answered]
I used the automated Azure Deployment with a little help from a youtube video. Everything worked from the first try and I'm currently using the cert I issued with the bot. 
The thing that's left unclear is how to keep the bot up to date, is there a way for CI/CD and is it adviseable?

**Comment by shibayan** [comment, marked as answer]
Acmebot always deploys the latest version each time it is started, so you do not need to worry about updates.
